src/Files.App/ServicesImplementation/QuickAccessService.cs

```diff
@@ -17,8 +17,10 @@ public async Task<List<string>> GetPinnedFoldersAsync()
 				.Where(link => link.IsFolder)
 				.Select(link => link.FilePath).ToList();
 
-			if (sidebarItems.Count > 4) // Avoid first opening crash #11139
-				sidebarItems.RemoveRange(sidebarItems.Count - 4, 4); // 4 is the number of recent items shown in explorer sidebar
+			// Avoid first opening crash #11139
+			if (sidebarItems.Count > 4)
+				// 4 is the number of recent items shown in explorer sidebar
+				sidebarItems.RemoveRange(sidebarItems.Count - 4, 4);
```

Comment:
You may consider adding a const like `const int RECENT_ITEMS_NUMBER = 4;` to make it even more readable

Reply:
That was me lol but yes, should be more readable that way

Reply:
This code was deleted by newly commits